contrib/gopass/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "gopass"
+pkgver = "1.15.11"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+pkgdesc = "Slightly more awesome standard UNIX password manager for teams"
```

Comment:
pkgdescs should be terse, descriptive, and not subjective; i.e. what is "awesome" is not decided by the author but by users; "standard" is highly debatable; etc.

Reply:
I changed it to 'standard unix password manager for teams'.
Password-store has the description 'standard unix password manager', so that should be good.

Reply:
that should be changed then

Reply:
is advanced console-based password manager good?

Reply:
presumably just "console-based password manager", but then they would both have the same pkgdesc. do we care about them being unique?

Reply:
pass could be "simple console-based password manager" cause gopass has more features

Reply:
i don't really care much about them being unique, maybe `Pass-compatible password manager with more features` or something

Reply:
that sounds good, i've updated the pkgdesc to that